net/xia/ppal_ether/main.c

```diff
@@ -18,12 +20,132 @@ static inline struct xip_ether_ctx *ctx_ether(struct xip_ppal_ctx *ctx)
 		: NULL;
 }
 
+/* ETHER-INTERFACE. */
+struct ether_interface {
+	struct net_device  *dev;
+	atomic_t           refcnt;
+	struct rcu_head    rcu_head;
+
+	/* Prevents racing conditions over list. */
+	spinlock_t         interface_lock;
+	struct list_head   list_interface_common_addr;
+};
+
+static inline struct ether_interface *__ether_get_rtnl(
+		const struct net_device *dev)
+{
+	return rtnl_dereference(dev->eth_ptr);
+}
+
+static inline struct ether_interface *ether_interface_get(
```

Comment:
Drop the `inline` tag.

Reply:
But won't the execution be faster if it is inline as it is just 9 lines?